charts/pulsar-operator/templates/bookkeeper-operator/operator_rbac.yaml

```diff
@@ -0,0 +1,67 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+{{- if .Values.components.bookkeeper }}
+kind: {{ template "pulsar.bookkeeperRoleKind" . }}
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: '{{ template "pulsar.fullname" . }}-{{ .Values.bookkeeper.rbac.name }}-operator'
+  {{- if not .Values.bookkeeper.serviceAccount.clusterRole }}
+  namespace: {{ template "pulsar.namespace" .}}
+  {{- end }}
+  labels:
+    {{- include "pulsar.standardLabels" . | nindent 4 }}
+    components: {{ .Values.bookkeeper.component }}
+rules:
+- apiGroups:
```

Comment:
is the permission enough for operator to run?

Reply:
> is the permission enough for operator to run?

It looks good now, I tried to install a release of sn-platform, but in the meantime didn't find another permission error in operator's log.